packages/react-storage/src/components/StorageManager/ui/FileList/FileList.tsx

```diff
@@ -25,7 +26,7 @@ export function FileList({
   const headingMaxFiles = getMaxFilesErrorText(maxFileCount);
 
   return (
-    <View className={'amplify-storagemanager--filelist'}>
+    <View className={ComponentClassNames.StorageManagerFileList}>
```

Comment:
I was just about to fix this! Thank you! But wait, I don't see `ComponentClassNames` being imported? NM its fixed now